package.json

```diff
@@ -49,7 +49,7 @@
     "classnames": "^2.1.5",
     "debug": "^2.2.0",
     "lodash": "^4.6.1",
-    "react-portal": "^2.2.1",
+    "react-dom": "^15.3.1",
```

Comment:
`ReactDOM` is required by the `Portal` component:
```js
    this.portal = ReactDOM.unstable_renderSubtreeIntoContainer(
      this,
      child,
      this.node
    )
```

Is there a way we can get around that?